### ISSUE TITLE
[TypingIndicator] Issue: Indicator no longer shows up on guilds

### ISSUE THREAD

**Issue by leem919** [l0c4lh057/BetterDiscordStuff#110]
_**Before you submit an issue please restart your discord. If the issue is still there feel free to write a bug report.**_

**Describe the bug**
With the guild indicator enabled, it doesn't show up at all anymore.

**To Reproduce**
1. Enable indicator on guilds
2. Watch for typing on guilds.

**Expected behavior**
Typing indicator shows up on guilds when someone types in it.

**Screenshots**
None.

**Information (please complete the following information):**
 - OS: Windows 10
 - Plugin Version: 0.5.3
 - Discord version: Stable
 - Compact Mode enabled? no

**Additional context**
None.

**Comment by Xzer0o0** [comment]
i have the same problem, didn't work with me

**Comment by marcpinet** [comment]
@l0c4lh057

**Comment by l0c4lh057** [comment]
> l0c4lh057

Yes, it is Christmas, soon to be new year, I'm currently visiting my family and will see a friend for new year. That is my favourite time to fix plugins.

(I know this has been an issue for quite some time but the reminder could have been timed better. Also I already have a WIP fix for it on my computer, however I won't be on my computer for another week.)

**Comment by marcpinet** [comment]
> > l0c4lh057
> 
> Yes, it is Christmas, soon to be new year, I'm currently visiting my family and will see a friend for new year. That is my favourite time to fix plugins.
> 
> (I know this has been an issue for quite some time but the reminder could have been timed better. Also I already have a WIP fix for it on my computer, however I won't be on my computer for another week.)

Okay thanks !

**Comment by XxUnkn0wnxX** [comment]
not sure if this is related but getting errors in console 

![Screen Shot 2022-01-23 at 5 58 19 am](https://user-images.githubusercontent.com/2242880/150651902-87669225-e257-4f22-837d-59bfb4d16999.jpg)


```text
53f28003fa5e1976babf.js:130 [TypingIndicator] Could not find Guild components
(anonymous) @ 53f28003fa5e1976babf.js:130
(anonymous) @ VM417:4
patchGuildList @ TypingIndicator.plugin.js:255
onStart @ TypingIndicator.plugin.js:185
start @ 0PluginLibrary.plugin.js:4352
startPlugin @ VM417:4
startAddon @ VM417:4
loadAddon @ VM417:4
loadAddon @ VM417:4
loadAllAddons @ VM417:4
initialize @ VM417:4
initialize @ VM417:4
startup @ VM417:4
async function (async)
startup @ VM417:4
(anonymous) @ VM417:4
n @ VM417:4
(anonymous) @ VM417:4
(anonymous) @ VM417:4
(anonymous) @ VM417:9
e.startsWith.e.startsWith.WebFrame.<computed> @ VM75 renderer_init.js:89
(anonymous) @ VM75 renderer_init.js:117
(anonymous) @ VM75 renderer_init.js:101
emit @ VM14 events.js:315
onMessage @ VM75 renderer_init.js:93
```
